docker/build-rpms.sh

```diff
@@ -89,7 +91,12 @@ else
    CASSANDRA_REVISION="${dt}git${ref}"
 fi
 
-if [ $JAVA_VERSION = "11" ]; then
+if [ $JAVA_VERSION = "17" ]; then
+   sudo alternatives --set java $(alternatives --display java | grep 'family java-17-openjdk' | cut -d' ' -f1)
+   sudo alternatives --set javac $(alternatives --display javac | grep 'family java-17-openjdk' | cut -d' ' -f1)
+   export CASSANDRA_USE_JDK17=true
```

Comment:
`CASSANDRA_USE_JDK17` currently exists, and is required, off @ekaterinadimitrova2 branch.

It will get removed, pre- or post- merge, depending if @ekaterinadimitrova2 includes the`JdkProperties` approach in the `netty-upgrade-jdk17test` branch. (Otherwise it will be short lived and removed when [18133](https://issues.apache.org/jira/browse/CASSANDRA-18133) lands.)

Reply:
That branch is WIP and those things were just a switch to JDK17 for testing long ago.
I don't mind the switch after we cleared that this is fine with the community (there were discussions). The opposite - all in :-) Simplification is always a good thing

Reply:
> I don't mind the switch to JdkProperties after we cleared that this is fine with the community (there were discussions)

we got agreement.